code/game/objects/items/devices/instruments.dm

```diff
@@ -164,3 +202,68 @@
 	throw_range = 15
 	hitsound = 'sound/items/bikehorn.ogg'
 
+//spooky stuff
+/obj/item/device/instrument/brass/trumpet/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/saxophone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/saxophone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/trombone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/Cn4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+//spooky procs
+/obj/item/device/instrument/brass/proc/spectral_attack(mob/living/carbon/C, mob/user)
+	if(ishuman(user)) //this weapon wasn't meant for mortals.
+		var/mob/living/carbon/human/U = user
+		if(!istype(U.dna.species, /datum/species/skeleton))
+			U.adjustStaminaLoss(25) //Extra Damage
+			attack(user)
+			//U.Knockdown(30)
+			to_chat(U, "<span class= 'danger'> Your ears weren't meant for this spectral sound.</span>")
+			return ..()
+
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(istype(H.dna.species, /datum/species/skeleton))
+			return ..() //undead are unaffected by the spook-pocalypse.
+		if(istype(H.dna.species, /datum/species/zombie))
+			H.adjustStaminaLoss(25)
+			H.Knockdown(15) //zombies can't resist the doot
+		C.Jitter(35)
+		C.stuttering = 20
+		C.adjustStaminaLoss(25) //only humanoids lose the will to live
+		to_chat(C, "<font color='red' size='4'><B>DOOT</B></span>")
+		var/T = get_turf(C)
+		if((H.getStaminaLoss() > 95) && (!istype(H.dna.species, /datum/species/skeleton)) && (!istype(H.dna.species, /datum/species/golem)) && (!istype(H.dna.species, /datum/species/android)) && (!istype(H.dna.species, /datum/species/jelly)))
+			H.Knockdown(20)
+			H.set_species(/datum/species/skeleton)
+			H.visible_message("<span class='warning'>[H] has given up on life as a mortal.</span>")
+			if(too_spooky)
+				if(prob(30))
+					new/obj/item/device/instrument/brass/saxophone/spectral(T)
+				else if(prob(30))
+					new/obj/item/device/instrument/brass/trumpet/spectral(T)
+				else if(prob(30))
+					new/obj/item/device/instrument/brass/trombone/spectral(T)
+				else
+					to_chat(H, "The spooky gods forgot to ship your instrument. Better luck next unlife.")
+			to_chat(H, "<B>You are the spooky skeleton!</B>")
+			to_chat(H, "A new life and identity has begun. Help your fellow skeletons into bringing out the spooky-pocalypse. You haven't forgotten your past life, but you are no longer beholden to any past loyalties.")

```

Comment:
What happened? You reverted it to the old text. I assume a admin/coder told you that it didn't matter?

Reply:
Wow. I was certain I changed that. I guess I may have reverted it by accident, or forgot to copy-paste that part over.